src/doc/style-guide/src/README.md

```diff
@@ -99,6 +99,42 @@ fn bar() {}
 fn baz() {}
 ```
 
+### Sorting
+
+In various cases, the default Rust style specifies to sort things. If not
+otherwise specified, such sorting should be "version sorting", which ensures
+that (for instance) `x8` comes before `x16` even though the character `1` comes
+before the character `8`. (If not otherwise specified, version-sorting is
+lexicographical.)
+
+For the purposes of the Rust style, to compare two strings for version-sorting:
+
+- Compare the strings by (Unicode) character as normal, finding the index of
+  the first differing character. (If the two strings do not have the same
+  length, this may be the end of the shorter string.)
+- For both strings, determine the longest sequence of ASCII digits that either
+  contains or ends at that index. (If either string doesn't have such a
+  sequence of ASCII digits, fall back to comparing the strings as normal.)
```

Comment:
What does "normal" refer to here?

Reply:
Lexicographically, as in the first bullet. I can say that explicitly.